src/Bridge/Doctrine/DoctrineDataStore.php

```diff
@@ -31,7 +32,7 @@ public function find(string $type, $id): ?object
         return $this->getEntityManager()->getRepository($type)->find($id);
     }
 
-    public function getPaginated(string $type, int $page, FilterCollectionInterface $filters, $maxPerPage = 30): PagerFanta
+    public function getPaginated(string $type, int $page, FilterCollectionInterface $filters, $maxPerPage = 30, PagesRequest $pagesRequest = null): PagerFanta
```

Comment:
This is a useless BC break.

Reply:
I added this against a BC actually... Why a nullable last parameter on an interface causes a fatal error? It's stupid to me right now

Reply:
@mcsky 
It's not stupid, it's breaking because:
1. This bc break is useless because as I said the datastore should not alter the pager
2. It is a bc break because an interface is designed to be implemented (unless it's tagged `@internal`). See [it in action here](https://3v4l.org/c95AB).
3. Please be kind, I'm not that stupid :) (got it, you was saying the new parameter is stupid because it's not necessary for datastore, that's what I was saying by saying "useless")